### ISSUE TITLE
usermanager load error in Chrome / OSX

### ISSUE THREAD

**Issue by lguariento** [eXist-db/usermanager#12]
The usermanager just throws a 'server responded: 404 - Not Found' error upon loading the usermanager. The console shows:

```
existdb-usermanager-app connected
index.html:21883 GET http://myip/exist/apps/usermanager/api/user/ 404 (Not Found) [index.html:21883]
GET http://myip/exist/apps/usermanager/api/group/ 404 (Not Found) [index.html:21883]
Error: The request failed with status code: 404  at HTMLElement.<anonymous> (index.html:21810)
```

The network console shows that `http://myip/exist/apps/usermanager/api/user/` and `http://myip/exist/apps/usermanager/api/group/` are not found (404). Refreshing the cache and logging out and in again doesn't solve the issue.

Chrome 64.0.3282.167 on OSX 10.13.3, eXist/db 4.0.0, usermanager 0.15.1

**Comment by duncdrum** [comment]
Can you check your dashboard version. It should be 1.0.0

**Comment by JoernT** [comment]
@duncdrum not that we're confusing things. Dashboard 1.0.0 is the new one which is not yet released. eXistdb 3.7.x still should be used with the old one.

**Comment by duncdrum** [comment]
The latest release on the public repo for exist 4.0.0 is dashboard 1.0.0. 
[public repo](http://demo.exist-db.org/exist/apps/public-repo/packages/dashboard.html?eXist-db-min-version=3.7.0-SNAPSHOT) if this is wrong we need to take it down.

**Comment by joewiz** [comment]
Dashboard v1.0.0 was released with eXist 4.0.0. This was discussed and agreed to at https://github.com/eXist-db/exist/issues/1729. If there is a new version of Dashboard in the works, then it should be released as the next major version - presumably 2.0.0.

**Comment by joewiz** [comment]
@lguariento Please see the release notes for 4.0.0, which contain directions for overcoming upgrade problems with Dashboard.

**Comment by lguariento** [comment]
@joewiz Yes, I saw them and acted accordingly. Besides, I also did a brand new install of the new eXist-db. The dashboard otherwise works well, the issue is with usermanager.

**Comment by lguariento** [comment]
I should probably also mention that I'm running exist-db on port 80 and without the trailing /exist in <Set name="contextPath">.

**Comment by adamretter** [comment]
Also reported here - https://github.com/eXist-db/exist/issues/1735

**Comment by maartenligtvoet** [comment]
I think this is not isolated to OSX
I have this issue also when running eXist on Linux CentOS 7.
We configure the contextPath in jetty.xml as / instead of the default /exist

**Comment by lguariento** [comment]
I meant OSX client-side, I'm also deploying eXist-db on CentoOS 7.

**Comment by JoernT** [comment]
should be fixed with 0.15.4
